src/app/portlets/dot-edit-page/shared/services/dot-edit-page-resolver/dot-edit-page-resolver.service.spec.ts

```diff
@@ -173,7 +173,7 @@ describe('DotEditPageResolver', () => {
             dotPageStateServiceRequestPageSpy.and.returnValue(of(mock));
 
             resolver.resolve(route).subscribe((state: DotPageRenderState) => {
-                expect(state).toEqual(mock);
+                expect(state).toBeNull();
```

Comment:
@fmontes please help me to double check this, according to the logic in the service should be null